modules/bookCollection.js

```diff
@@ -0,0 +1,90 @@
+import { loadBooksFromStorage, saveBooksToStorage } from './storage.js';
+import {
+  displayBooks,
+  addBook,
+  removeBook,
+  showAddSection,
+  showListSection,
+  showContactSection,
+} from './bookUtilits.js';
+
+class BookCollection {
+  constructor() {
+    this.booksCollection = [];
+    this.loadBooksFromStorage();
+    this.displayBooks();
+
+    this.addLink = document.getElementById('add-link');
+    this.listLink = document.getElementById('list-link');
+    this.contactLink = document.getElementById('contact-link');
+    this.addSection = document.getElementById('add-section');
+    this.listSection = document.getElementById('list-section');
+    this.contactSection = document.getElementById('contact-section');
+    this.form = document.getElementById('form');
+    this.addButton = document.querySelector('.add-button');
+
+    this.addLink.addEventListener('click', this.showAddSection.bind(this));
+    this.listLink.addEventListener('click', this.showListSection.bind(this));
+    this.contactLink.addEventListener('click', this.showContactSection.bind(this));
+    this.addButton.addEventListener('click', this.addBook.bind(this));
+  }
+
+  loadBooksFromStorage() {
+    this.booksCollection = loadBooksFromStorage();
+  }
+
+  saveBooksToStorage() {
+    saveBooksToStorage(this.booksCollection);
+  }
+
+  displayBooks() {
+    displayBooks(
+      this.booksCollection,
+      this.removeBook.bind(this),
+    );
+  }
```

Comment:
- [ ]  You have done perfect work 😺, however, let's talk about one point. When adding books, everything is perfect then when click the `Booklist` button, I can see the list as shown the the image below:

![Screenshot 2023-06-06 153437](https://github.com/Innocent259/Awesome-Books-v2/assets/57408419/37eca84c-5f5e-4e21-b3e9-280d788d427a)

Then when we refresh the page, we are not able to see the book list as we can see in the image below : 

![Screenshot 2023-06-06 153535](https://github.com/Innocent259/Awesome-Books-v2/assets/57408419/457eaa3a-8a0f-4d0d-88ed-34998e767bfd)

let add a new book `Javascript` by `joel`, then if we go back to the list, from there we can see now all books wit the new `Javascript` book 

![Screenshot 2023-06-06 155051](https://github.com/Innocent259/Awesome-Books-v2/assets/57408419/edd23fac-9db9-4103-9fee-56ddc991d856)

Kindly, fix it 💪